<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMain</class>
 <widget class="QWidget" name="frmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tcp通信服务器端</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>491</width>
     <height>49</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="txtPort">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>60000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnListen">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>监听</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>端  口：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckHexSend">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>十六进制发送</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckHexReceive">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>十六进制接收</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>491</width>
     <height>425</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>已连接客户端：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtReceive">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>331</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSend">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>384</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSend">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>384</y>
      <width>341</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cboxClient">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>350</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>清空数据</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
